pkg/state/helmx.go

```diff
@@ -26,6 +26,23 @@ func (st *HelmState) appendHelmXFlags(flags []string, release *ReleaseSpec) ([]s
 	return flags, nil
 }
 
+// append post-renderer flags to helm flags
+// nolint: unparam
+func (st *HelmState) appendPostRenderFlags(flags []string, release *ReleaseSpec, helm helmexec.Interface) ([]string, error) {
```

Comment:
I think there is no need to return error here

Reply:
Similar cases will be unified pr for optimization